app/facades/foods_facade.rb

```diff
@@ -0,0 +1,8 @@
+class FoodsFacade
+  def self.find_foods(food)
+    json = FoodsService.search_foods(food)
+    count = json[:totalHits]
+    foods = json[:foods].map { |food_data| Food.new(food_data, count) }
+    ten_foods = foods.shift(10)
```

Comment:
Very nice! I really like that you're getting 10 food results back, as well as totalHits, with just one API call.
I think it would be sufficient to return `count` and `ten_foods` from this method, rather than storing the count on each PORO. One drawback of storing the count this way is that it will take up space in local memory, but that would probably only come up as a concern on a much larger project.